class/book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rent(person, date)
+    rental = Rental.new(person, self, date)
+    @rentals << rental
+    person.rentals << rental
+    rental
```

Comment:
- [Optional] Hi @Rishi-Mishra0704, great efforts for adding a method that will enable the user to rent out a book :+1: Instead of creating an object variable and returning it, I think we can even reduce these links of code to:
```ruby
  def rent(person, date)
    Rental.new(date, self, person)
  end
```

- What is your suggestion?